webpack.config.js

```diff
@@ -0,0 +1,76 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+
+
+module.exports = {
+
+    context: path.resolve(__dirname, 'src'),
+
+    entry: [
+        'react-hot-loader/patch',
+        './index.jsx'
+    ],
+
+    output: {
+        filename: 'bundle.js',
+        path:  path.resolve(__dirname, 'build'),
+    },
+
+    resolve: {
+        extensions: ['.js', '.jsx']
+    },
+
+    devServer: {
```

Comment:
usually dev server configuration is moved to another folder/file. It's just webpack config